webui/react/config-overrides.js

```diff
@@ -15,7 +15,6 @@ const webpack = require('webpack');
 const jestConfig = require('./jest.config');
 
 const IS_DEV = process.env.NODE_ENV === 'development';
-const SERVER_ADDRESS = IS_DEV ? 'http://localhost:8080' : process.env.SERVER_ADDRESS;
```

Comment:
we can't do this for all react builds because https://github.com/determined-ai/determined/blob/b86fd4031442c906891d95079244a9f944fab6d4/webui/react/src/routes/utils.ts#L21 would then never be undefined which has meaning in our code

Reply:
I see, thanks for correcting and explaining this portion.